examples/siro_sandbox/server/messaging_service.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Dict, List
+
+
+class MessagingService:
```

Comment:
I wish we had a more meaningful name. Maybe ClientMessagingService? To convey that this is particularly for sending messages to the remote client. And maybe ClientMessageManager? I chose "SandboxService" to perhaps convey that it doesn't particularly own or manage the things it offers, whereas clearly this class owns and manages these messages.